css/properties/user-modify.json

```diff
@@ -0,0 +1,123 @@
+{
+  "css": {
+    "properties": {
+      "user-modify": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/user-modify",
+          "support": {
+            "chrome": {
+              "prefix": "-webkit-",
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "prefix": "-webkit-",
+              "version_added": "18"
+            },
+            "edge": {
+              "prefix": "-webkit-",
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "prefix": "-moz-",
+              "version_added": true,
+              "partial_implementation": true,
+              "notes": "While the CSS property is parsed and accepted, it does not have any effect."
```

Comment:
Not sure this is actually true, but we should mention that it is planned to be removed in https://bugzilla.mozilla.org/show_bug.cgi?id=1388910.

Reply:
Well, [that’s it currently says on MDN](https://developer.mozilla.org/docs/Web/CSS/user-modify#Browser_compatibility).